ogr/services/github/project.py

```diff
@@ -359,8 +359,13 @@ def fork_create(self) -> "GithubProject":
         :return: fork GithubProject instance
         """
         gh_user = self.github_instance.get_user()
-        fork = gh_user.create_fork(self.github_repo)
-        return GithubProject("", self.service, "", github_repo=fork)
+        fork: Repository = gh_user.create_fork(self.github_repo)
+        project = GithubProject("", self.service, "", github_repo=fork)
```

Comment:
Can we use `None` instead of `""`? And ideally, keyword arguments so it's clear what's what?

Reply:
done